### ISSUE TITLE
Allow for different formats of files to be used.

### ISSUE THREAD

**Issue by Mohamed-Ayaan358** [acmpesuecc/Doc-Scan#2]
### What do we want you to do?

Pdf files and word files at the moment can be used in the script but more files like .pptx, .jpg(Compare the binary encoding of the jpg so as to see if they differ), and .mp4

### Possible solutions

Look into other libraries that can also compare for example these libraries were taken from elsewhere and after sufficient tweaking, they were made to fit in. You can perform audio analysis on mp4 files to compare the two of them.

#### 1. Open Access time: 
 [1 hour] if each block time has passed

#### 2. Block time: 
[1 hour], to show significant progress.

**Comment by samarth777** [comment]
can you assign this to me

**Comment by Mohamed-Ayaan358** [comment]
Yes @samarth777 You may remember the block time is an hour so  give an update by then.

**Comment by samarth777** [comment]
yup I will

**Comment by samarth777** [comment]
I am done with the code for image files

**Comment by samarth777** [comment]
bro, for ppt files, if I just tell which slide is different, it;s fine right

**Comment by Mohamed-Ayaan358** [comment]
Yes that will be quite good @samarth777, do remember it has to be compatible with the tkinter module that has been merged in.

**Comment by samarth777** [comment]
yup I will make sure to do that

**Comment by samarth777** [comment]
I am done with pptx and image files, do I send a pull request or shall I do it after mp4 also?

**Comment by Mohamed-Ayaan358** [comment]
If you can do mp4 that would be good.

**Comment by samarth777** [comment]
I will send this much as a PR for now okay?

**Comment by Mohamed-Ayaan358** [comment]
Sure go ahead, it works with tkinter right?

**Comment by Mohamed-Ayaan358** [comment]
And also screenshots

**Comment by samarth777** [comment]
oh I should only integrate with the gui also?

they are seperate functions rn
that take input from the terminal
but it's fine I will integrate with GUI and send PR if needed

**Comment by Mohamed-Ayaan358** [comment]
Yes you must intergrate with the GUI

**Comment by ad1tya24** [comment]
I would like to work on it
